racunaljka.js

```diff
@@ -0,0 +1,54 @@
+function Body(length) {
+    this.length = length;
+}
+
+Body.prototype.growth = function(newLength) {
+    this.length += newLength;
+}
+
+function Head(left, right) {
+    this.leftEye = left;
+    this.rightEye = right;
+}
+
+Head.prototype.changeColor = function(color) {
+    this.leftEye = color.leftEye;
+    this.rightEye = color.rightEye;
+}
+
+function Tail(leftPosition, center, rightPosition) {
+    this.leftPosition = leftPosition;
+    this.center = center;
+    this.rightPosition = rightPosition;
+}
+
+Tail.prototype.moving = function(move) {
+    this.leftPosition = move.rightPosition;
+    this.center = move.centar;
+    this.rightPosition = move.leftPosition;
+}
+
+function Snake(kind) {
+    this.kind = kind;
+    this.length = new Body(3);
```

Comment:
The naming is wrong, it is completely unintuitive, you should be using class names or similar:
```
this.head = new Head('green', 'green');
this.body = new Body(3);
this.tail = new Tail('', 'center', '');
```